### PR TITLE
Changes made during the deployment on the dev environment

### PR DESCRIPTION
**Pull request by jluk-codi**

### CODE REVIEW

File: hiera/common.yaml [hunk 1]
```diff
@@ -64,5 +64,4 @@ zuul::connections:
     driver: 'github'
     sshkey: '/var/lib/zuul/ssh/id_rsa'
 
-zuul::zookeeper_hosts:
-  - "zuulv3.opencontrail.org:2181"
+zuul::zookeeper_hosts: "zuulv3-dev.opencontrail.org:2181"
```
Comment (kklimonda) [inline hiera/common.yaml:67]: This should be a list, in case we deploy a zookeeper cluster

File: manifests/site.pp [hunk 2]
```diff
@@ -23,18 +23,13 @@
 }
 
 node 'zuulv3-dev.opencontrail.org' {
+  class { '::opencontrail_ci::server': }
   class { '::opencontrail_ci::zuul_scheduler': }
-  class { '::zuul::web': }
-  class { '::zuul::executor': }
+  #class { '::zuul::executor': }
```
Comment (kklimonda) [inline manifests/site.pp]: remove commented out code

**Review by kklimonda — CHANGES_REQUESTED** [review]
Also, try keeping git commit messages in a format similar to (for example) https://github.com/erlang/otp/wiki/Writing-good-commit-messages (short summary under 50-60 characters and longer description after that). Various tools expect summary to be short, and hide parts when it gets too long.

File: hiera/common.yaml (hunk 1 above)
Comment (jluk-codi) [reply hiera/common.yaml:67]: the yaml list format gets rendered in the zuul.conf ini file as json and breaks the configuration. It has to be a comma-delimited string if we want to specify multiple hosts

**Review by kklimonda — CHANGES_REQUESTED** [review]
```
~/c/g/k/contrail-infra ❯❯❯ rake syntax && rake lint
---> syntax:manifests
---> syntax:templates
---> syntax:hiera:yaml
modules/opencontrail_ci/manifests/zuul_merger.pp:2 [ERROR] trailing whitespace found
```

**Comment by jluk-codi** [comment]
fixed

**Pull request merged**